src/contentTypeConfig.js

```diff
@@ -4,6 +4,8 @@ let contentTypeConfig = {
             '"artist_name": artist->name',
             'soundslice',
             'instrumentless',
+            '"id": railcontent_id',
+            '"type": _type',
```

Comment:
These fields are part of the `defaultFields` object, why are they needed here?

Reply:
defaultFields is defined only in fetchAll method, not globally